docs/user-guide/kubeconfig-file.md

```diff
@@ -94,7 +94,89 @@ users:
     client-key: path/to/my/client/key
 ```
 
-### Building your own kubeconfig file
+### Breakdown/explanation of components
+
+#### cluster
+
+```
+clusters:
+- cluster:
+    api-version: v1
+    server: http://cow.org:8080
+  name: cow-cluster
+```
+
+A `cluster` contains endpoint data for a kubernetes cluster. This includes the fully
+qualified url for the kubernetes apiserver, as well as the cluster's certificate
+authority or `insecure-skip-tls-verify: true`, if the cluster does not support tls.
```

Comment:
Suggest "... certificate authority specified in `certificate-authority`, or `insecure-skip-tls-verify: true` if the cluster does not support TLS."

Perhaps include `certificate-authority` and/or `insecure-skip-tls-verify: true` in the example here would make it more clear?